### ISSUE TITLE
Script to export all images at full resolution, and create an SVG containing them

### ISSUE THREAD

**Issue by jywarren** [publiclab/Leaflet.DistortableImage#164]
Building on the full-resolution export completed in https://github.com/publiclab/Leaflet.DistortableImage/pull/100, we should think of a way to export all images (it might need to be one at a time or it'd overload the system?) and then combine them all into a single document in SVG format. 

This would actually save space and processing power vs. combining them into a giant raster image. 

We could use this library: 

https://svgjs.com/docs/2.7/elements/#svg-image

The syntax looks pretty simple; 

```js
var draw = SVG('drawing').size(300, 300)
var image = draw.image('/path/to/image.jpg', 200, 300)
```

We'd love help with this and it could be an interim solution in parallel to the exporter work going on in https://github.com/publiclab/Leaflet.DistortableImage/issues/126! @sashadev-sky, what do you think of this?

**Comment by sashadev-sky** [comment]
@jywarren yeah I would love to work on this one! Should I get started here (serverless export) instead of mapknitter?

**Comment by jywarren** [comment]
sure!!

**Comment by rexagod** [comment]
@sashadev-sky Are you working on this (or have plans to)?

**Comment by sashadev-sky** [comment]
Plan to in the future! Working on the exporter from the mapknitter side first here: #413. If you end up wanting to work on it before I get to it that's totally fine, just mention me here to notify me

**Comment by jywarren** [comment]
Also connecting to PDF generation idea here: https://github.com/publiclab/mapknitter/issues/219

**Comment by jywarren** [comment]
So, also noting this would be a great function of `DistortableCollection` - PDF or SVG.
